charts/vastcsi/values.yaml

```
@@ -63,6 +63,8 @@ storageClassDefaults:
   mountOptions: []
   # Name of QoS policy associates with the view.
   qosPolicy: ""
+  # Reclaim policy to use with the storage class.
+  reclaimPolicy: ""
```

@aveuiller - we're thinking to make this `Delete`, since that would be the default behavior provided by the storage-class when this attribute is missing. what do you think?

```suggestion
  reclaimPolicy: "Delete"
```

---

That is also the default value coming from kubernetes. Having it explicitly defined as delete sounds good though, I don't see any issue with that.